lib/ClimaLandSimulations/src/utilities/met_drivers_FLUXNET.jl

```diff
@@ -0,0 +1,224 @@
+    precipitation_function(t::FT) where {FT} =
+        p_spline(t) < 0.0 ? p_spline(t) : 0.0 # m/s
+    snow_precip(t) = FT(0) # this is likely not correct
+
+    # Construct the drivers. For the reference time we will use the UTC time at the
+    # start of the simulation
+    atmos = ClimaLSM.PrescribedAtmosphere(
+        precipitation_function,
+        snow_precip,
+        atmos_T,
+        atmos_u,
+        atmos_q,
+        atmos_p,
+        UTC_DATETIME[1],
+        atmos_h;
+        c_co2 = atmos_co2,
+    )
+
+    function zenith_angle(
+        t,
+        ref_time;
+        latitude = lat,
+        longitude = long,
+        insol_params::Insolation.Parameters.InsolationParameters{FT} = earth_param_set.insol_params,
+    ) where {FT}
+        # This should be time in UTC
+        current_datetime = ref_time + Dates.Second(round(t))
+
+        # Orbital Data uses Float64, so we need to convert to our sim FT
+        d, δ, η_UTC =
+            FT.(
+                Insolation.helper_instantaneous_zenith_angle(
+                    current_datetime,
+                    ref_time,
+                    insol_params,
+                )
+            )
+
+        FT(
+            Insolation.instantaneous_zenith_angle(
+                d,
+                δ,
+                η_UTC,
+                longitude,
+                latitude,
+            )[1],
+        )
+    end
+
+    radiation = ClimaLSM.PrescribedRadiativeFluxes(
+        FT,
+        SW_IN_spline,
+        LW_IN_spline,
+        UTC_DATETIME[1];
+        θs = zenith_angle,
+    )
+
+    # Start and end dates of data in MODIS format
+    modis_start_date = "A$(Dates.year(UTC_DATETIME[1]))$(lpad(Dates.dayofyear(UTC_DATETIME[1]), 3, "0"))"
+    modis_end_date = "A$(Dates.year(UTC_DATETIME[end]))$(lpad(Dates.dayofyear(UTC_DATETIME[end]), 3, "0"))"
+
+    MODIS_LAI = single_col_data_matrix(
+        parse_response(
+            check_response(
+                send_get_subset(
+                    "MCD15A2H",
+                    modis_start_date,
+                    modis_end_date,
+                    site_ID,
+                    band = "Lai_500m",
+                ),
+            ),
+        ),
+    )
+
+    LAI_dt = Second(MODIS_LAI[2, 1] - MODIS_LAI[1, 1]).value
+    LAI_seconds = FT.(0:LAI_dt:((length(MODIS_LAI[:, 1]) - 1) * LAI_dt))
+
+    # LAI spline for radiative transfer
+    LAIspline = Spline1D(LAI_seconds, MODIS_LAI[:, 2])
+    LAIfunction = (t) -> eltype(t)(LAIspline(t))
+
+    # Necessary inputs from LAI Data
+    # Note that f_root_to_shoot, capacity, and h_leaf are site-specific parameters
+    # defined in the parameters file for each site
+    maxLAI = FT(maximum(MODIS_LAI[:, 2]))
+    RAI = maxLAI * f_root_to_shoot
+    plant_ν = capacity / (maxLAI * h_leaf) / FT(1000)
+
+    return (
+        LOCAL_DATETIME,
+        atmos_co2,
+        DATA_DT,
+        drivers,
+        atmos,
+        radiation,
+        LAIfunction,
+        maxLAI,
```

Comment:
would it be possible to define max LAI, RAI and plant \nu somewhere else? they arent really "drivers", more parameters.

Reply:
I think this is up to discussion, since we read LAI data. Eventually LAI and RAI will be prognostic with carbon pools. Agreed for plant_\nu though. Not sure what to do yet